Cargo.toml

```diff
@@ -16,22 +17,17 @@ name = "good_stv_server"
 path = "src/bin/server/main.rs"
 
 [dependencies]
-clap = "2.30.0"
-csv = "1.0.0-beta.5"
-env_logger = "0.5.3"
-failure = "0.1.1"
-log = "0.4.1"
-rand = "0.4.2"
-rocket = "0.3.6"
-rocket_codegen = "0.3.6"
-serde = "1.0"
-serde_derive = "1.0"
-serde_json = "1.0"
-slog = "2.1.1"
```

Comment:
No more slog?

Reply:
The problem I was using slog to work around is now gone afaik, so no more slog :). It's honestly a pain in the ass.

Reply:
this sound familiar.